src/AdminConsole/Helpers/TimeSpanExtensions.cs

```diff
@@ -0,0 +1,16 @@
+namespace Passwordless.AdminConsole.Helpers;
+
+public static class TimeSpanExtensions
```

Comment:
Move this to the `Common` project. We might need it elsewhere.

Reply:
This is unused. Removing instead.